modules/search/pom.xml

```diff
@@ -87,6 +87,15 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpcore-osgi</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
```

Comment:
Isn't all of `httpclient` contained in `httpclient-osgi`?

Reply:
I have no clue. The dependency analyzer has given me some trouble saying I didn't declare that dependency. The CI build does the same now for the `admin-ui` and I have no idea why 😞 .